pyswmm/tests/test_rainfallAPI.py

```diff
@@ -0,0 +1,39 @@
+from pyswmm import Simulation, RainGages, SystemStats
+from pyswmm.tests.data import MODEL_RAIN
+
+def test_api_rainfall():
+    with Simulation(MODEL_RAIN) as sim:
+        check = 0
+
+        sim._model.setGagePrecip("Gage1", 10.00)
+        for ind, step in enumerate(sim):
+            sim._model.setGagePrecip("Gage1", 10.00)
```

Comment:
can rain gage precip be set during model simulation

Reply:
Yes, we fan inject our own hyetograph during the simulation ;-) that's why this feature is baller.